### PR TITLE
Mention metric changes in vespa 8

### PR DESCRIPTION
**Pull request by baldersheim**
@yngveaasheim @vekterli @gjoranv PR
@geirst FYI

### CODE REVIEW

File: en/vespa8-release-notes.html [hunk 1]
```diff
@@ -873,6 +874,39 @@ <h3 id="vespa-http-client">vespa-http-client</h3>
...
 
 
+<h2 id="renamed-metrics">Renamed metrics</h2>
+<p>
+The following metrics are renamed:
+</p>
+<table class="table">
+  <thead>
+    <tr>
+      <th>Old Name</th><th>Name name</th><th>Description</th>
```
Comment (vekterli) [inline en/vespa8-release-notes.html:884]: s/Name name/New name/

Comment (gjoranv) [inline en/vespa8-release-notes.html:884]: "Name name" => "New name"

File: en/vespa8-release-notes.html [hunk 2]
```diff
@@ -873,6 +874,39 @@ <h3 id="vespa-http-client">vespa-http-client</h3>
...
+      <td><em>sum</em> has been removed from the metric name.</td>
+    </tr>
+    <tr>
+      <td><em>vds.filestor.*.sum.*</em></td>
+      <td>vds.filestor.*.*</td>
+      <td><em>sum</em> has been removed from the metric name.</td>
+    </tr>
+    <tr>
+      <td><em>vds.distribtor.*.sum.*</em></td>
+      <td>vds.distribtor.*.*</td>
```
Comment (vekterli) [inline en/vespa8-release-notes.html:905]: s/distribtor/distributor for these two lines

File: en/vespa8-release-notes.html [hunk 3]
```diff
@@ -893,6 +927,90 @@ <h2 id="removed-metrics">Removed metrics</h2>
...
+    <tr>
+      <td><em>vds.distributor.updates.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.gets.sum.latency.average</em></td>
+      <td>Use .sum/.count instead</td>
+    </tr>
+    <tr>
+      <td><em>vds.distributor.visitor.sum.latency.average</em></td>
```
Comment (vekterli) [inline en/vespa8-release-notes.html:1011]: These all seem to use the legacy metric names (including `.sum.`). Should we use specify the new ones instead? Or both?

**Review by vekterli — APPROVED** [review]
:+1: a few minor questions/comments

File: en/vespa8-release-notes.html [hunk 4]
```diff
@@ -893,6 +927,90 @@ <h2 id="removed-metrics">Removed metrics</h2>
       <td><em>http.status.403.rate</em></td>
       <td>Use <em>http.status.4xx.rate</em> with dimension <em>statusCode</em>==403</td>
     </tr>
+    <tr>
+      <td><em>vds.visitor.allthreads.averagevisitorlifetime.sum.average</em></td>
```
Comment (gjoranv) [inline en/vespa8-release-notes.html:931]: Perhaps use strikethrough on 'sum.average' to mark that the '.sum' part is also gone? Some of these metrics are also renamed, right?

**Review by gjoranv — APPROVED** [review]

File: en/vespa8-release-notes.html (hunk 1 above)
Comment (baldersheim) [reply en/vespa8-release-notes.html]: Fixed

Comment (baldersheim) [reply en/vespa8-release-notes.html]: Fixed

File: en/vespa8-release-notes.html (hunk 2 above)
Comment (baldersheim) [reply en/vespa8-release-notes.html]: Fixed

**Comment by gjoranv** [comment]
Any other comments, or should we just merge this?

File: en/vespa8-release-notes.html (hunk 4 above)
Comment (baldersheim) [reply en/vespa8-release-notes.html]: I have use strikethrough on all elements that will be removed.
That is .sum and .alldisks

File: en/vespa8-release-notes.html (hunk 3 above)
Comment (baldersheim) [reply en/vespa8-release-notes.html]: I think that using strikethrough should be clear enough.
If anyone has used any of these metrics, they are very likely to infer the action required.

**Comment by gjoranv** [comment]
👍

**Pull request merged**